qa/tasks/cephfs/mount.py

```diff
@@ -573,18 +573,6 @@ def mounted_wait(self):
         finally:
             self.umount_wait()
 
-    def is_blocklisted(self):
-        addr = self.get_global_addr()
-        if addr is None:
-            log.warn("Couldn't get the client address, so the blocklisted status undetermined")
-            return False
-
-        blocklist = json.loads(self.fs.mon_manager.raw_cluster_cmd("osd", "blocklist", "ls", "--format=json"))
```

Comment:
Why not just create a MonManager for each mount in `__init__`?

Reply:
That can be done too...

Reply:
Originally I avoided creating it because all commands in mount classes were commands meant for client machine. Creating a CephManager instance would not be simple since __init__() is run by mount objects of vstart_runner as well. vstart_runner objects use instances of LocalCephManager instead. 

I am moving the original patch back here. The "new CephManager instance" patch lives here - https://github.com/rishabh-d-dave/ceph/tree/fs-qa-mon_manager-2.

Reply:
very well